### PR TITLE
gsagh-366

### PR DESCRIPTION
**Pull request by DominikaLos**

### CODE REVIEW

File: GsaGH/Parameters/5_Results/GsaNodeDisplacement.cs [hunk 1]
```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GsaGH.Parameters._5_Results {
```
Comment (kpne) [inline GsaGH/Parameters/5_Results/GsaNodeDisplacement.cs:7]:
```suggestion
namespace GsaGH.Parameters {
```

File: GsaGH/Parameters/5_Results/GsaNodeDisplacement.cs [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  public class GsaNodeDisplacement {
+    internal Dictionary<string, GsaResultsValues> ACaseResultValues { get; private set; }
+      = new Dictionary<string, GsaResultsValues>();
+
+    internal void AddACaseValue(string nodelist, GsaResultsValues values) {
```
Comment (kpne) [inline GsaGH/Parameters/5_Results/GsaNodeDisplacement.cs:12]:
```suggestion
    internal void AddAnalysisCaseValues(string nodelist, GsaResultsValues values) {
```

File: GsaGH/Parameters/5_Results/GsaResult.cs [hunk 3]
```diff
@@ -7,6 +7,7 @@
 using GsaAPI;
 using GsaGH.Helpers;
 using GsaGH.Helpers.GsaApi;
+using GsaGH.Parameters._5_Results;
```
Comment (kpne) [inline GsaGH/Parameters/5_Results/GsaResult.cs:10]: see comment above

File: GsaGH/Parameters/5_Results/GsaResult.cs [hunk 4]
```diff
@@ -139,8 +140,8 @@ public class GsaResult {
     ///   Append to this dictionary to chache results
     ///   key = elementList
     /// </summary>
-    internal Dictionary<string, GsaResultsValues> ACaseNodeDisplacementValues { get; set; }
-      = new Dictionary<string, GsaResultsValues>();
+    internal GsaNodeDisplacement ACaseNodeDisplacementValues { get; set; }
```
Comment (kpne) [inline GsaGH/Parameters/5_Results/GsaResult.cs:142]:
```suggestion
    internal GsaNodeDisplacement AnalysisCaseNodeDisplacements { get; set; }
```

File: GsaGH/Parameters/5_Results/GsaNodeDisplacement.cs [hunk 5]
```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GsaGH.Parameters._5_Results {
+  public class GsaNodeDisplacement {
```
Comment (kpne) [inline GsaGH/Parameters/5_Results/GsaNodeDisplacement.cs:8]:
```suggestion
  public class GsaNodeDisplacements {
```
File name should be changed as well

File: GsaGH/Parameters/5_Results/GsaResult.cs [hunk 6]
```diff
@@ -931,17 +932,17 @@ public class GsaResult {
...
-          ACaseNodeDisplacementValues.Add(nodelist,
+          ACaseNodeDisplacementValues.AddACaseValue(nodelist,
             ResultHelper.GetNodeResultValues(ACaseNodeResults[nodelist], lengthUnit));
         }
 
         return new Tuple<List<GsaResultsValues>, List<int>>(new List<GsaResultsValues> {
-          ACaseNodeDisplacementValues[nodelist],
+          ACaseNodeDisplacementValues.ACaseResultValues[nodelist],
         }, Model.Model.Nodes(nodelist).Keys.ToList());
       }
```
Comment (kpne) [inline GsaGH/Parameters/5_Results/GsaResult.cs]: Should we move this to a method in the new class?

Comment (DominikaLos) [reply GsaGH/Parameters/5_Results/GsaResult.cs]: not a 1 step I suppose

File: GsaGH/Parameters/5_Results/GsaNodeDisplacements.cs [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GsaGH.Parameters {
+  public class GsaNodeDisplacements {
+    internal Dictionary<string, GsaResultsValues> ACaseResultValues { get; private set; }
+      = new Dictionary<string, GsaResultsValues>();
+
+    internal void AddAnalysisCaseValue(string nodelist, GsaResultsValues values) {
```
Comment (kpne) [inline GsaGH/Parameters/5_Results/GsaNodeDisplacements.cs:12]:
```suggestion
    internal void AddAnalysisCaseValues(string nodelist, GsaResultsValues values) {
```

**Pull request closed without merging**